swagger_parser/swagger_parser.py

```diff
@@ -791,7 +801,14 @@ def get_response_example(self, resp_spec):
                 definition_name = self.get_definition_name_from_ref(resp_spec['schema']['$ref'])
                 return self.definitions_example[definition_name]
             elif 'items' in resp_spec['schema'] and resp_spec['schema']['type'] == 'array':  # Array
-                definition_name = self.get_definition_name_from_ref(resp_spec['schema']['items']['$ref'])
+                if '$ref' in resp_spec['schema']['items']:
+                    definition_name = self.get_definition_name_from_ref(resp_spec['schema']['items']['$ref'])
+                else:
+                    if 'type' in resp_spec['schema']['items']:
```

Comment:
this is the only time you check if 'type' is in resp_spec['schema']['items'], all other times you assume it's there